verification/src/accept_block.rs

```diff
@@ -180,7 +180,7 @@ impl<'a> BlockCoinbaseMinerReward<'a> {
 				let prevout = store.transaction_output(&input.previous_output, tx_idx);
 				let (sum, overflow) = incoming.overflowing_add(prevout.map(|o| o.value).unwrap_or(0));
 				if overflow {
-					return Err(Error::ReferencedInputsSumOverflow)
+					return Err(Error::Transaction(tx_idx, TransactionError::Overspend));
```

Comment:
Isn't it overflow and not overspend? Won't distinguishing between overflow and actual overspend help a little ?